VimeoNetworkingExample-iOS/VimeoNetworkingCommonTests/VIMVideoTests.swift

```diff
@@ -49,4 +49,43 @@ class VIMVideoTests: XCTestCase
         XCTAssertNotNil(testVideoObject)
         XCTAssertFalse(testVideoObject.isLive())
     }
+    
+    func test_isLiveEventInProgress_returnsTrue_whenEventIsInMidBroadcastState()
+    {
+        let liveDictionary: [String: Any] = ["link": "vimeo.com", "key": "abcdefg", "activeTime": Date(), "endedTime": Date(), "archivedTime": Date(), "status": "ready"]
+        let testLiveObject = VIMLive(keyValueDictionary: liveDictionary)!
```

Comment:
Let's safe-unwrap this variable.

Reply:
@supperspidey It should be fine to force unwrap here since we're trying to ensure that everything is working as expected. 

If anything in these tests fails, including object construction, then we know something has gone 👻 _horribly_ 👻 wrong and can investigate. If the video object or live object fails to unwrap then I'd waaay rather it result in a failed test instead of unwrapping safely or executing an alternate code path.